Command/HealthCheckCommand.php

```diff
@@ -51,18 +51,24 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
         $runner->useAdditionalReporters($input->getOption('reporter'));
 
+        if (null !== $checkNames = $input->getArgument('checkNames')) {
+            $checkNames = explode(',', $checkNames);
```

Comment:
Please be more defensive here. I could provide "test1, test2,     test3" which would evaluate as "test1", " test2", "     test3", which is not what I want.